### ISSUE TITLE
File. Keyboard Navigation Issue

### ISSUE THREAD

**Issue by mikeamelin** [surveyjs/survey-library#6136]
Pressing Enter doesn't work.

![Image](https://user-images.githubusercontent.com/17025812/236124851-9df8f8f5-5580-41b4-ad54-3b4ca7cc1f09.png)

**Comment by novikov82** [comment]
See private messages for more info...

**Comment by novikov82** [comment]
If you cancel the download of the picture (press ESC), then the focus moves somewhere to the beginning of the application. Also, the button itself still remains as if focused.

**Comment by novikov82** [comment]
As soon as the image is uploaded, the second image upload dialog immediately appears.
And so on ad infinitum if you use the keyboard (confirm image loading by enter). If you click the Upload button in the dialog, then everything is fine.
This only applies to Safari.
Everything is fine in Chrome.
![image](https://github.com/surveyjs/survey-library/assets/29385489/6add3307-3575-47a3-9103-4174cae95c30)

**Comment by novikov82** [comment]
New issue opend for this case: https://github.com/surveyjs/survey-library/issues/6199
